README.md

```diff
@@ -689,7 +689,7 @@ by adding `@include oGridSurfaceLayoutSizes;` to your Sass file.
 
 ### Markup/Sass changes
 
-- Wrap top-level `<div class="o-grid-row">` into `<div class="o-grid-container">`.
+- Wrap `<div class="o-grid-container">` around any `<div class="o-grid-row">`.
```

Comment:
The "top-level" needs to remain clear as it's only the top-level `<div class="o-grid-row">` that needs wrapping. The change could instead be "inside" instead of "into"?